apps/web/src/components/lectureCard/LectureCard.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react"
+import styles from "./lectureCard.module.scss"
+import { Card } from "./card/LecturePersonCard"
+import { Text } from "@/components/text/Text"
+import Image from "next/image"
+
+import avatar from "@/assets/Rectangle 225.svg"
+import time from "@/assets/Access time.svg"
```

Comment:
nie widziałem tych assetów na gicie, jeśli ich nie ma to trzeba je dodać

osobna rzecz to nazwa tych assetów, zmieńmy to na coś pokroju default-avatar.svg i time.svg, spacje w nazwach jak teraz nie są ok :/